src/AchievementService/AchievementService.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+

```

Comment:
maybe you need "<Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />" here and in libraries, consult Mariya